### DISCUSSION TITLE
Thumbnails of images in overview list and entry detail where bigger (adjustable in size)

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9388] [Ideas]
> Created by: Tim Bertens (tim@bene.be) on 2015/09/10 09:31:18 +0000
> Votes at time of UserVoice import: 4

-----------------------------------

Make the thumbnails of images appear much larger (or adjustable in size) in the control panel entries (both in the detail of the entry and in the overview list using the new customisable columns in 2.5). 

The standard preview of the images is so small it hardly is visible to control panel users.

**Comment by angrybrad** [comment]
> Posted by Tim Bertens (tim@bene.be) on 2015/09/10 09:31:18 +0000
-----------------------------------

Ok, found out already that in the entry detail view the thumbnail can made (a bit) larger since 2.5.  Would be better if the size would be configurable though.  Still have a valid use case for the image size in the entries overview pages

**Comment by theskyfloor** [comment]
A slider like macOS would be awesome!

<img width="140" alt="screen shot 2017-01-31 at 2 15 35 pm" src="https://cloud.githubusercontent.com/assets/8887839/22482727/e26c50d0-e7bf-11e6-90da-381764be4dd6.png">

**Comment by MattAppleton** [comment]
Just to say we are very keen on seeing bigger thumbnails in entries – any chance this can be rolled into v4?

Out users are used to Prismic / Wordpress etc where we can offer much more of a 'visual' listing for things like video entries, where Craft's small thumbnails end up all looking very similar due to the scale.


As per @angrybrad:  "The standard preview of the images is so small it hardly is visible to control panel users."

**Comment by josh-attwood** [comment]
Also would be very interested in seeing this.

We have a few sites that this would be really helpful for on the client side. The size of the thumbnails are pretty unhelpful as they are at the minute.

**Comment by fermincreative** [comment]
Just adding my support for this idea here - I have a client who's managing a catalogue of unique items with very similar names. Properly sized thumbnails would make a world of a difference to them!

**Comment by sm9** [comment]
It would be great if this was possible in Craft 4. I typically have entries with a 'Hero Image' field, and it would be nice if there was a View Mode that was bigger than the current 'Large Thumbnail' view. A configurable size per entry type, or an option for the thumbnail to fill the space/width taken up by the field (25%/50%/75%100%) would be even better as there's a lot of wasted space at the moment where I would prefer to see a larger image - [Screenshot.](https://capture.dropbox.com/fKSaq9MCXrGebvMc)
